src/java.base/share/classes/java/lang/Double.java

```diff
@@ -143,6 +143,55 @@
  * elements of a {@link java.util.SortedSet SortedSet} or as keys of a
  * {@link java.util.SortedMap SortedMap}.
  *
+ * <p>While {@code ==} is not an equivalence relation, several useful
+ * equivalence relations can be defined over floating-point
+ * values. Comparing numerical equality to various kinds of
+ * equivalence:
+ *
+ * <dl>
+ * <dt><i>numerical equality</i> ({@code ==} operator): (<em>Not</em>
+ * an equivalence relation)</dt>
+ * <dd>Two floating-point values represent the same extended real
+ * number. The extended real numbers are the real numbers augmented
+ * with positive infinity and negative infinity. Under numerical
+ * equality, {@code +0.0} and {@code -0.0} are equal since they both
+ * map to the same real value, 0. A NaN does not map to any real
+ * number and is not equal to any value, including itself.
+ * </dd>
+ *
+ * <dt><i>bit-wise equivalence</i>:</dt>
+ * <dd>The bits of the two floating-point values are the same. This
+ * equivalence relation for {@code double} values {@code a} and {@code b} is
+ * implemented by the expression
+ * <br>{@code Double.doubleTo}<code><b>Raw</b></code>{@code LongBits(a) == Double.doubleTo}<code><b>Raw</b></code>{@code LongBits(b)}<br>
```

Comment:
Place in a code snippet since it's hard to read in source otherwise? Snippets allow for highlighting e.g.
```java
/**
 * {@snippet :
 * Double.doubleRawToLongBits(a) == Double.doubleToRawLongBits(b) // @highlight substring="Raw"
 * }
 */
```

Reply:
Before sending out the PR, I used a snippet, but it formatted the code using too much vertical space for the definition list. I didn't look to see if there was a "compact" styling option that could be used.

Reply:
Ok, i am not aware of a compact style.

Reply:
Filed [JDK-8294814](https://bugs.openjdk.org/browse/JDK-8294814)